cockpit_api/src/test/java/com/cockpit/api/controller/UnassignTeamControllerTest.java

```diff
@@ -0,0 +1,69 @@
+package com.cockpit.api.controller;
```

Comment:
Is it better to name this script MvpControllerTest instead of UnassignTeamControllerTest? For the next, we will add other endpoints tests for mvp.

Reply:
ça marche